job-service-scheduled-executor/src/main/java/com/github/jobservice/scheduledexecutor/JobTaskData.java

```diff
@@ -37,6 +39,8 @@ public class JobTaskData
     private String taskPipe;
     @NotNull
     private String targetPipe;
+    @Null
```

Comment:
This means `null`-only, not nullable.  Just get rid of it.  Nullable is the default.